portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -50,8 +52,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     
     UserService userService = UserServiceFactory.getUserService();
     List<Task> comments = new ArrayList<>();
-    if (userService.isUserLoggedIn()) {
 
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    String uploadUrl = blobstoreService.createUploadUrl("/new-comment");
```

Comment:
Minor nit here - I would put "/new-comment" into a variable of some sort. Definitely when we're working on a project with multiple people, having this in a variable someplace will be more useful than having it as a hard coded magic constant.